packages/framework/src/Resources/views/Admin/Content/Order/orderItem.html.twig

```diff
@@ -18,9 +18,19 @@
             {{ form_widget(orderItemForm.catnum, {attr: { class: 'input--medium text-right'}}) }}
             {{ form_errors(orderItemForm.catnum, {errors_attr: { inline: false }}) }}
         </td>
-        <td class="text-right">
-            {{ self.priceWithVatWidget(orderItemForm.unitPriceWithVat) }}
-        </td>
+
+        {% if inputPriceType is same as(constant('Shopsys\\FrameworkBundle\\Model\\Pricing\\PricingSetting::INPUT_PRICE_TYPE_WITH_VAT')) %}
+            <td class="text-right">
+                {{ self.inputPriceWidget(orderItemForm.unitPriceWithVat) }}
+            </td>
+            <td class="text-right">{{ self.calculablePriceWidget(orderItemForm.unitPriceWithoutVat) }}</td>
+        {% elseif inputPriceType is same as(constant('Shopsys\\FrameworkBundle\\Model\\Pricing\\PricingSetting::INPUT_PRICE_TYPE_WITHOUT_VAT')) %}
+            <td class="text-right">
+                {{ self.calculablePriceWidget(orderItemForm.unitPriceWithVat) }}
```

Comment:
`unitPriceWithVat` should be used here always?

Reply:
I do not understand the question. `calculablePriceWidget` and `inputPriceWidget` are changing by INPUT_PRICE_TYPE here.